.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*       @shopify/merchant-and-partner-experience
+*       @Shopify/pp-partners
```

Comment:
Github linter is erroring because this team is not publicly visible, however the syntax is valid and should serve our desired purpose of being added to PR reviews